charts/astronomer/values.yaml

```diff
@@ -24,11 +24,11 @@ images:
     # httpSecret: ~
   houston:
     repository: quay.io/astronomer/ap-houston-api
-    tag: 0.26.9
+    tag: release-0.27
```

Comment:
0.27.0?

Reply:
As per CI logs `docker push astronomerinc/ap-astro-ui:release-0.27`

Reply:
both 0.27.0 and release-0.27 exist in https://quay.io/repository/astronomer/ap-houston-api?tab=tags.

0.27.0 is explicitly this first version on the release branch and release-0.27 will be the tag published whenever the next houston build on release-0.27 branch happens.